### ISSUE TITLE
Cloud not find dictionary with dotnet script

### ISSUE THREAD

**Issue by tani** [kekyo/MeCab.DotNet#4]
This issue is related to #3. When we use dotnet5 new feature `#r "nuget: ..."` syntax in F#, 
the runtime could not find the dictionary. We can reproduce the following steps,

```
$ cat test.fsx
#r "nuget: MeCab.DotNet"
open System
open MeCab

let input = "行く川のながれは絶えずして、しかももとの水にあらず。"
let tagger = MeCabTagger.Create (MeCabParam())
input
    |> tagger.ParseToNodes
    |> Seq.filter (fun node -> node.CharType > 0u)
    |> Seq.map (fun node -> node.Feature.Split ',')
    |> Seq.iter (fun features -> printf "%s" features.[7])
$ dotnet fsi test.fsx
System.IO.DirectoryNotFoundException: Could not find a part of the path '/Users/masaya/.nuget/packages/mecab.dotnet/0.0.30/lib/net5.0/dic/char.bin'.
   at Interop.ThrowExceptionForIoErrno(ErrorInfo errorInfo, String path, Boolean isDirectory, Func`2 errorRewriter)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String path, OpenFlags flags, Int32 mode)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access)
   at MeCab.Core.CharProperty.Open(String dicDir)
   at MeCab.Core.Tokenizer.Open(MeCabParam param)
   at MeCab.Core.Viterbi.Open(MeCabParam param)
   at MeCab.MeCabTagger.Open(MeCabParam param)
   at MeCab.MeCabTagger.Create(MeCabParam param)
   at <StartupCode$FSI_0002>.$FSI_0002.main@()
```

I don't know whether C#-script is also so or not. It might be so, I guess.

**Comment by kekyo** [comment]
Thank you feedback, I understood what problem it.

fsi (and other without project source code?) will not place any `Content` fragments from MSBuild project declaration.
MeCab.DotNet handles default dict files inside project statically including directive.

Hmm, will think how to fix it.

**Comment by tani** [comment]
One of the naive solutions is embedding dictionary data as the base64 string.
The hard coded data is not possible to update easily, but it is enough to work I guess.

**Comment by kekyo** [comment]
@tani Fixed in 0.0.40 version. Could you test it?

**Comment by tani** [comment]
Look at this, it works well!

```
$ vi test.fsx
$ cat test.fsx
#r "nuget: MeCab.DotNet"
open System
open MeCab

let input = "行く川のながれは絶えずして、しかももとの水にあらず。"
let tagger = MeCabTagger.Create (MeCabParam())
input
    |> tagger.ParseToNodes
    |> Seq.filter (fun node -> node.CharType > 0u)
    |> Seq.map (fun node -> node.Feature.Split ',')
    |> Seq.iter (fun features -> printf "%s" features.[7])
$ dotnet fsi test.fsx
イクカワノナガレハタエズシテ、シカモモトノミズニアラズ
```

Thank you for fantastic update!
